modules/virt-jumbo-frames-vm-pod-nw.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc              
+
+:_content-type: CONCEPT                                            
+[id="virt-jumbo-frames-vm-pod-nw_{context}"]                                   
+= About jumbo frames support          
+
+When using the OVN-Kubernetes CNI plug-in, you can send unfragmented jumbo frame packets between two virtual machines (VMs) that are connected on the default pod network. Jumbo frames have a maximum transmission unit (MTU) value greater than 1500 bytes.
```

Comment:
"plugin" rather than "plug-in"

https://redhat-documentation.github.io/supplementary-style-guide/#grammar